src/MCPClient/lib/clientScripts/failedSIPCleanup.py

```diff
@@ -22,9 +22,10 @@ def main(fail_type, sip_uuid):
     models.SIPArrange.objects.filter(sip_id=sip_uuid).delete()
 
     # Update storage service that reingest failed
-    api = storage_service._storage_api()
+    session = storage_service._storage_api_session()
+    url = storage_service._storage_service_url() + 'file/' + sip_uuid + '/'
```

Comment:
Different authors, I think. I agree that a good future improvement would be to move this to `storageService.py`